packages/api/src/modules/deposit-scdl-process/check/DepositScdlProcess.check.service.test.ts

```diff
@@ -0,0 +1,57 @@
+import { CreateDepositScdlLogDto, DepositScdlLogDto } from "dto";
+import depositScdlProcessCheckService from "./DepositScdlProcess.check.service";
+import { BadRequestError, ConflictError } from "core";
+import { DEPOSIT_LOG_PATCH_DTO_PARTIAL_STEP_2 } from "../__fixtures__/depositLog.fixture";
+
+describe("DepositScdlProcess check service", () => {
+    describe("validateCreate", () => {
+        it("should accept valid data", () => {
+            const dto: CreateDepositScdlLogDto = { overwriteAlert: true };
+
+            const fn = () => depositScdlProcessCheckService.validateCreate(dto);
+
+            expect(fn).not.toThrow();
```

Comment:
je trouve ça un peu lourd de définir une variable `fn`.

Pourquoi ne pas faire comme dans les autres tests et directement définir la méthode dans le expect ?

```suggestion
            expect(() => depositScdlProcessCheckService.validateCreate(dto)).not.toThrow();
```